tools/api-markdown-documenter/src/api-item-transforms/configuration/Hierarchy.ts

```diff
@@ -305,11 +304,10 @@ export namespace DefaultHierarchyConfigurations {
 	export function getDocumentName(apiItem: ApiItem, config: HierarchyConfiguration): string {
 		const kind = getApiItemKind(apiItem);
 		switch (kind) {
-			case ApiItemKind.Model: {
-				return "index";
-			}
+			case ApiItemKind.Model:
+			case ApiItemKind.Namespace:
 			case ApiItemKind.Package: {
-				return getUnscopedPackageName(apiItem as ApiPackage);
+				return "index";
```

Comment:
So this bit doesn't impact how the link is rendered, right?

Reply:
The only change would be from  `.../foo` to `.../foo/`